src/promptflow/promptflow/_sdk/_submitter/experiment_orchestrator.py

```diff
@@ -116,7 +116,8 @@ def _run_flow_node(self, node, experiment, run_dict):
             variant=node.variant,
             flow=node.path,
             connections=node.connections,
-            experiment_name=experiment.name,
+            # Experiment name can't work for now, it is used only for show cloud exp name
```

Comment:
maybe just remove this? as if we want to add back by using experiment.name? experiment is already an parameter to ExeprimentRun